### DISCUSSION TITLE
[Feature Request] Gradual Infill Steps

### DISCUSSION THREAD

**Discussion by Playfutbol2** [OrcaSlicer/OrcaSlicer#3228] [Ideas]
Cura has an excellent feature called "Gradual Infill Steps" that essentially gives the user control over adaptive infill. It will increase infill density the closer the print is to a top surface layer, improving top layer quality. Yes, adaptive infill does this but it is only useful in very large, thick prints. Gradual infill steps is much more useful in my opinion, allowing me to more easily have less infill density for most of a print but denser infill right underneath top surfaces.

**Comment by TrueBlueRoo** [comment]
I was just looking for that setting. I agree it is a nice feature buyt not known by many. It significantly reduces infill.

**Comment by psiberfunk** [comment]
There’s also a great 3d  version /idea called gradient infill that would serve a similar purpose ::   https://github.com/WatchingWatches/GradientInfill/tree/master

**Comment by psiberfunk** [comment]
Also idea maker has had progressive infill since 2020! I’m shocked it hasn’t made it into the Prusa family of slicers

**Comment by CyberGeneticist** [comment]
+1, this is a great feature in Cura that I often wish was in Orca. Please add if possible, thank you
